local/o365/classes/page/acp.php

```diff
@@ -1131,8 +1131,9 @@ public function mode_teamconnections_connect() {
                 $DB->insert_record('local_o365_objects', $grouprecord);
             }
 
-            $DB->delete_records('local_o365_objects', ['type' => 'group', 'subtype' => 'courseteam', 'moodleid' => $courseid]);
-            $DB->delete_records('local_o365_objects', ['type' => 'group', 'subtype' => 'teamfromgroup', 'moodleid' => $courseid]);
+            $DB->delete_records_select('local_o365_objects',
+                'type => "group" AND subtype IN ("courseteam", "teamfromgroup") AND moodleid = ?', [$courseid]);
```

Comment:
Presumably the `=>` here was intended to be a `=`. Several other instances in this commit also exist.
```
ERROR:  syntax error at or near "=>"
LINE 1: SELECT * FROM mdl_local_o365_objects WHERE type => "group" A...
                                                        ^
```

Reply:
@usqfowlerj Thank you so much for reporting the issue. We have made a new release containing the fix. Please check it out.